### PR TITLE
add post handler for 911 game script

### PR DESCRIPTION
**Pull request by Ryanlewis2018**

### CODE REVIEW

File: server/src/routes/global.ts [hunk 1]
```diff
@@ -12,6 +13,17 @@ router.get("/911-calls", async (req: IRequest, res: Response) => {
   return res.json({ calls, status: "success" });
 });
 
+router.post("/911-calls", async (req: IRequest, res: Response) => {
+	const id = v4();
+	const { location, description, caller } = req.body
```
Comment (Dev-CasperTheGhost) [inline server/src/routes/global.ts:18]: Can you add a check if the description was not provided? The database will throw an error if there's no description.

```js
let description = req.body.description || "No description provided"
```

**Review by Dev-CasperTheGhost — CHANGES_REQUESTED** [review]
Please check requested changes, for the rest it looks good!

**Comment by Ryanlewis2018** [comment]
Added the check and tested.

**Comment by Dev-CasperTheGhost** [comment]
Alright, will check out the updated code when back from school.

**Review by Dev-CasperTheGhost — APPROVED** [review]
Looks good

**Pull request merged**